test/UnitTests/MSTest.CoreAdapter.Unit.Tests/Execution/TestMethodRunnerTests.cs

```diff
@@ -429,9 +429,11 @@ public void RunTestMethodForMultipleResultsReturnMultipleResults()
             var testMethodRunner = new TestMethodRunner(testMethodInfo, this.testMethod, this.testContextImplementation, false);
```

Comment:
There are a bunch of data-driven tests in this class. Please make sure they are validating correct things even though they are not failing.

Reply:
Various data driven tests were failing. We have fixed them in the PR.